jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml

```diff
@@ -626,15 +622,23 @@
             version-cluster: '1.2'
             version-client: '1.3'
             version-infix: '1-2-1-3'
+            version-env: |
+                # 1.2 doesn't support IAM, so we should use cert auth.
```

Comment:
This one shouldn't be necessary. 1.3 client should be able to handle everything fine.